project-fortis-interfaces/src/components/Admin/Admin.js

```diff
@@ -136,7 +136,7 @@ class Admin extends React.Component {
                     <TabPanel>
                       <h2>Blacklisted Terms <FontIcon className="fa fa-question" data-tip data-for='blacklist' data-place='right' color={blue500} hoverColor={blue700}/></h2>
                       <ReactTooltip id='blacklist'>
-                        <span>Provide a comma delimited list of blacklist terms, i.e. hunger, flood, hurricane</span>
+                        <span>If an event matches all the blacklist terms, then it will be discarded.</span>
```

Comment:
Keep the example and comma-delimited hint that you had earlier. That definitely adds value -- I'd just mention both pieces of information.

Reply:
oh ok oops...I'll also squash and merge this so that I don't have so many commits.